lib/index.js

```diff
@@ -45,18 +41,25 @@ Embark = {
   },
 
   deployContracts: function(env, contractFiles, destFile, chainFile, withProvider, withChain, cb) {
+    var web3 = this.web3;
+    web3.setProvider(new web3.providers.HttpProvider("http://" + this.blockchainConfig.rpcHost + ":" + this.blockchainConfig.rpcPort))
```

Comment:
wouldn't be good to let the http:// on the config file so later on it would be easily switched to https if needed?